Report/CostBreakdownChart.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * @copyright   2019 Mautic Contributors. All rights reserved
+ * @author      Digital Media Solutions, LLC
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace MauticPlugin\MauticMediaBundle\Report;
+
+use Doctrine\Common\Cache\CacheProvider;
+use Doctrine\ORM\EntityManager;
+use MauticPlugin\MauticMediaBundle\Entity\StatRepository;
+
+class CostBreakdownChart
+{
+    /**
+     * Pretty colors ooooooo.
+     *
+     * @var array
+     */
+    private $providerColors = [
+        'facebook' => [
+            'backgroundColor'           => 'rgba(59,89,153 ,0.5)',
+            'borderColor'               => 'rgba(59,89,153 ,1)',
+            'pointHoverBackgroundColor' => 'rgba(59,89,153 ,0.5)',
+            'pointHoverBorderColor'     => 'rgba(59,89,153 ,0.5)',
+        ],
+        'snapchat' => [
+            'backgroundColor'           => 'rgba(255,252,0 ,0.5)',
+            'borderColor'               => 'rgba(255,205,0, 1)',
+            'pointHoverBackgroundColor' => 'rgba(255,252,0 ,0.5)',
+            'pointHoverBorderColor'     => 'rgba(255,252,0 ,0.5)',
+        ],
+        'bing' => [
+            'backgroundColor'           => 'rgba(0, 255, 0, 0.5)',
+            'borderColor'               => 'rgba(0, 144, 0, 1)',
+            'pointHoverBackgroundColor' => 'rgba(0, 255, 0, 0.5)',
+            'pointHoverBorderColor'     => 'rgba(0, 255, 0, 0.5)',
+        ],
+        'google' => [
+            'backgroundColor'           => 'rgba(221,75,57 ,0.5)',
+            'borderColor'               => 'rgba(221,75,57 ,1)',
+            'pointHoverBackgroundColor' => 'rgba(221,75,57 ,0.5)',
+            'pointHoverBorderColor'     => 'rgba(221,75,57 ,0.5)',
+        ],
+    ];
+
+    /**
+     * @var CostBreakdownReporter
+     */
+    private $reporter;
+
+    /**
+     * CostBreakdownChart's constructor.
+     *
+     * @param StatRepository     $statRepository
+     * @param EntityManager      $em
+     * @param CacheProvider|null $cache
+     */
```

Comment:
Does this need to be updated with current param list?

Reply:
Yes